### ISSUE TITLE
open JIRA for "no go"

### ISSUE THREAD

**Issue by vlimant** [CMSCompOps/WmAgentScripts#363]
when some workflows are held on go=False, a JIRA can be created automatically following the integration in #356

**Comment by vlimant** [comment]
we can think of having the campaign being set up from the JIRA opened this way. Unified can pick up the last comment with a valid json from that JIRA
@mweinberg2718 @sharad1126

**Comment by sharad1126** [comment]
Yes this would be great @vlimant I think I can work on this with @SvenjaPflitsch
